src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector.cs

```diff
@@ -85,6 +85,12 @@ public override BoundStatement InstrumentFieldOrPropertyInitializer(BoundStateme
 
         private static BoundStatement InstrumentFieldOrPropertyInitializer(BoundStatement rewritten, SyntaxNode syntax)
         {
+            if (syntax.IsKind(SyntaxKind.Parameter))
+            {
+                // This is an initialization of a generated property based on record parameter.
+                return AddSequencePoint(rewritten);
```

Comment:
Do we have a test to show the sequence point?

Reply:
We don't at the moment. We will have to test debugging support at some point, I think that should happen at that time. I can make an explicit note in Test Plan.

---
In reply to: [434399243](https://github.com/dotnet/roslyn/pull/44803#discussion_r434399243) [](ancestors = 434399243)

Reply:
BTW, this code path is getting hit by the existing tests. I just didn't add the test to verify presence of the sequence point.

---
In reply to: [434690200](https://github.com/dotnet/roslyn/pull/44803#discussion_r434690200) [](ancestors = 434690200,434399243)

Reply:
Yup, I understand. Added a note to test plan already (" verify sequence points"). Thanks